Cargo.toml

```diff
@@ -47,6 +47,10 @@ members = [
 [dependencies]
 # pins this version to fix https://github.com/actix/actix-net/issues/129
 actix-rt = "=1.1.1"
+openssl-probe = { version = "0.1.2"}
+
+[patch.crates-io]
+openssl-probe = { git = "https://github.com/khorolets/openssl-probe.git", branch="feature/support-macports" }
```

Comment:
is this supposed to be a long term solution?

Reply:
It is supposed to be removed once author of `openssl-probe` publish 0.2.0 version with fix, or fix the breaking change for 0.1.3